components/ShoppingItemList.js

```diff
@@ -0,0 +1,30 @@
+
+import ShoppingItemCard from "./ShoppingItemCard";
+import styled from "styled-components";
+export default function ShoppingItemList({ shoppingData }) {
+  return (
+    <>
+      <List>
+        {shoppingData.map((item) => {
+          return (
+            <li key={item._id}>
+              <ShoppingItemCard
+                name={item.name}
+                category={item.category}
+                quantity={item.quantity}
+                id={item._id}
+              />
+            </li>
+          );
+        })}
+      </List>
+    </>
```

Comment:
You don't need the fragment around the `List`
```suggestion
      <List>
        {shoppingData.map((item) => {
          return (
            <li key={item._id}>
              <ShoppingItemCard
                name={item.name}
                category={item.category}
                quantity={item.quantity}
                id={item._id}
              />
            </li>
          );
        })}
      </List>
```